modules/ROOT/pages/local_development.adoc

```diff
@@ -0,0 +1,147 @@
+include::{partialsdir}/attributes.adoc[]
+
+= Local Development
+
+To facilitate the development of mobile services on a users 
+local machine or trialling {org-name} {product-name} without requiring 
+extra hardware, it is possible to bring up {org-name} {product-name} on 
+your local machine. This guide takes you through the process 
+of setting up {org-name} {product-name} locally.
+
+== Requirements
+
+=== Ansible
+
+Ansible version 2.3 or later is a requirement of running the 
+installation scripts on your local machine, follow the 
+installation steps
+link:https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[here].
+
+=== Node.js
+
+Node.js version 8 or later and the bundled package manager 'npm' 
```

Comment:
is this actually still needed? I know we had it for when the UI was an extension, but what is it used for now?